src/gc/transaction_level_gc_manager.cpp

```diff
@@ -81,13 +84,13 @@ void TransactionLevelGCManager::Running(const int &thread_id) {
 }
 
 
-void TransactionLevelGCManager::RecycleTransaction(std::shared_ptr<GCSet> gc_set, const cid_t &timestamp) {
```

Comment:
Why did we use timestamp to hash the context? I think I may be the one who wrote this.. but I cannot remember why I did that..